apps/myaccount/src/pages/account-security.tsx

```diff
@@ -143,6 +144,21 @@ const AccountSecurityPage = (props: RouteProps): ReactElement => {
                         </Grid.Row>
                     ) : null }
 
+                { hasRequiredScopes(accessConfig?.security, accessConfig?.security?.scopes?.read, allowedScopes) &&
```

Comment:
Start the code on a new line after open curly brace. Also have the above ending curly brace in a new line.

Reply:
Fix this issue in other instances as well.